core/sql/cli/Cli.cpp

```diff
@@ -9611,12 +9611,28 @@ Lng32 SQLCLI_LOBddlInterface
 	    
 	  } // for
 
+        //Initialize LOB interface 
+        
+        Int32 rc= ExpLOBoper::initLOBglobal(exLobGlob,currContext.exHeap(),&currContext,hdfsServer,hdfsPort);
+        if (rc)
+          {
+            {
```

Comment:
It doesn't hurt anything, but I'm curious: Why two sets of braces? Looks like one would do?